package.json

```diff
@@ -54,6 +55,7 @@
     "image-size": "^0.5.1",
     "inquirer": "^3.0.6",
     "loader-utils": "^1.1.0",
+    "lodash": "^4.17.5",
```

Comment:
If you really need to use `merge` from `lodash`, use `lodash.merge` module - https://www.npmjs.com/package/lodash.merge

Reply:
Damn, I've bet that Satya will write this. XD